### DISCUSSION TITLE
Unable to connect to MongoDB

### DISCUSSION THREAD

**Discussion by kodecrash** [Budibase/budibase#6412] [Help] [answered]
I have configured MongoDB on **mongodb://localhost:27017** 

Below is my configuration in budibase

![image](https://user-images.githubusercontent.com/8296994/174828399-89be6c1e-6394-4143-834f-79766fb9dc02.png)

Getting below error once I click on Run Query

![image](https://user-images.githubusercontent.com/8296994/174828604-aae23425-3179-48c2-9e2a-2b55b44ec2f2.png)

**Comment by melohagan** [comment, marked as answer]
HI @kodecrash 

Some thoughts: 

1. Are you self hosting? If so you could try `host.docker.internal` instead of localhost
2. Is that port definitely exposed in your local instance - is the local instance within a docker container?
3. Do you definitely not need to provide a username and password in the connection string?

**Reply by kodecrash** [reply]
Solution first worked I am using docker container. Thanks.
